### PR TITLE
eks-d support

### PR DESCRIPTION
**Pull request by jrryjcksn**

### CODE REVIEW

File: pkg/flavors/eksd/eksd.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	n := strings.ToLower(name)
+	for _, c := range e.release.Status.Components {
+		if strings.ToLower(c.Name) == n {
+			return &c
+		}
+	}
+	return nil
+
+}
+func overrideKubeadm() (string, string, error) {
```
Comment (palemtnrider) [inline pkg/flavors/eksd/eksd.go:65]: Please add an issue to track this so we can replace this when upstream is fixed.

File: pkg/plan/recipe/install_plans.go [hunk 2]
```diff
@@ -184,10 +185,38 @@ func BuildCRIPlan(ctx context.Context, criSpec *existinginfrav1.ContainerRuntime
...
+	if f != nil {
+		log.Debugf("Using flavor %+v", f)
+		return func(binName, version string) plan.Resource {
+			// TODO (Mark) logic for the architecture
+			binURL, sha256, err := f.KubeBinURL(binName)
+			if err != nil {
+				log.Fatalf("%v", err)
+				return nil
+			}
+			// TODO Use the sha256 value to verify the downlaod
```
Comment (palemtnrider) [inline pkg/plan/recipe/install_plans.go:199]: Obsolete comment as we are checking the sha in the command below

**Review by palemtnrider — APPROVED** [review]
LGTM - just a nit and request for a new issue tracking the kubeadm override.

File: pkg/flavors/eksd/eksd.go (hunk 1 above)
Comment (jrryjcksn) [reply pkg/flavors/eksd/eksd.go:65]: Done

**Pull request merged**